packages/plugin-ext/src/plugin/tasks/tasks.ts

```diff
@@ -109,6 +109,21 @@ export class TasksExtImpl implements TasksExt {
         return this.createDisposable(callId);
     }
 
+    async executeTask(task: theia.Task): Promise<theia.TaskExecution> {
+        const taskDto = converter.fromTask(task);
+        if (taskDto) {
+            const executionDto = await this.proxy.$executeTask(taskDto);
+            if (executionDto) {
+                const taskExecution = this.getTaskExecution(executionDto);
+                return taskExecution;
+            }
+        }
+        return {
```

Comment:
maybe we should reject request for execution if a we can not run task for some reason
WDYT?